digits/config/store_option.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import config_option
+from . import prompt
+
+class ModelStoreOption(config_option.Option):
+
+    @staticmethod
+    def config_file_key():
+        return 'model_store'
+
+    @classmethod
+    def prompt_title(cls):
+        return 'Model Store URL list'
+
+    @classmethod
+    def visibility(cls):
+        return config_option.Visibility.HIDDEN
+
+    def optional(self):
+        return True
+
+    def suggestions(self):
+        self.url_list = ['http://localhost/modelstore/']
+        return [prompt.Suggestion(self.url_list, 'H', desc='URL list')]
+
+    def apply(self):
+        self._config_dict_value = {
+            'url_list': self.url_list
+        }
+
+
```

Comment:
I think I will implement it later in this PR.  The code should ensure that URL is supposed to be well-formatted, not random strings.